src/schemas/room.ts

```diff
@@ -17,6 +23,7 @@ export interface RoomPayload {
   memberIdList : string[],
   ownerId?     : string,
   adminIdList  : string[],
+  type?        : RoomType,
```

Comment:
The `type` name makes me feel too generic.

How about the name `external?: boolean`?

Reply:
Agree